### ISSUE TITLE
Case sensitive view sub-folders

### ISSUE THREAD

**Issue by randallmeeker** [balderdashy/sails#2342]
I know there were fixes to view pages, but what about sub folders? 
https://github.com/balderdashy/sails/pull/840

I just ran into this with the latest builds (using jade). I'm dev'ing on a Windows machine and pushed to a linux box using Git. 

```
- api
    - controllers
        - admin
            TrainingCenterController.js#view
                  .... return res.ok({data}) .....
- views
   - admin
        - TrainingCenter
            view.jade
```

When I pushed this, the url /admin/trainingcenter/view would only return JSON. 

When I changed ```TrainingCenter``` to ```trainingcenter``` it worked fine.

Either a bug, or needs a note in the Docs? I can doc it, if its intended behavior. 

For others that hit my exact problem, I needed to delete the subfolders (and contents) and add them back (via restore in recycle bin), because GIT did not care that I renamed them lower case.

**Comment by CWyrtzen** [comment]
Hi @randallmeeker did you ever find a resolution for this?

**Comment by randallmeeker** [comment]
Its up to the core if they think its a bug or not

**Comment by sailsbot** [comment]
Thanks for posting, @randallmeeker.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
